frontend/src/components/settings/pages/general/StoreSelect.tsx

```diff
@@ -11,17 +12,23 @@ const logger = new Logger('StoreSelect');
 const StoreSelect: FunctionComponent<{}> = () => {
   const [selectedStore, setSelectedStore] = useSetting<Store>('store', Store.Default);
   const [selectedStoreURL, setSelectedStoreURL] = useSetting<string | null>('store-url', null);
+  const { t } = useTranslation();
+  const tBranches = [
```

Comment:
Again, this is for store selection, not update channel.

Reply:
Also fixed this.